File: exercises_java/src/test/java/sandbox/HelloworldTest.java
package sandbox;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class HelloworldTest {

    @Test
    @DisplayName("サンプル")
    void hello() {
        Helloworld sut = new Helloworld();
        assertEquals("Hello World.", sut.hello());
    }
}

File: kotlin-multi-projects/core/src/test/kotlin/com/example/core/HelloWorldTest.kt
package com.example.core

import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.assertEquals

class HelloWorldTest {

    @Test
    @DisplayName("サンプル")
    internal fun hello() {
        val actual = HelloWorld().message()
        assertEquals("Hello World.", actual)
    }
}
